crates/rethnet_eth/src/remote/methods.rs

```diff
@@ -0,0 +1,15 @@
+use crate::remote::MethodInvocation as EthMethodInvocation;
+
+/// RPC methods specific to Hardhat
+pub mod hardhat;
+
+/// an RPC method with its parameters
+#[derive(Clone, Debug, PartialEq, serde::Deserialize, serde::Serialize)]
+#[serde(untagged)]
+#[allow(clippy::large_enum_variant)]
+pub enum MethodInvocation {
```

Comment:
hardhat methods are not part of `eth`. They are our own custom methods. As such, they and `MethodInvocation` should live outside of the `rethnet_eth` crate. They were fine where they were, or if you want in a separate `rethnet_rpc` crate

Reply:
i moved them back to `rethnet_rpc_server` in https://github.com/NomicFoundation/hardhat/pull/3998/commits/37b288e819d30d5913a8cb7425bc47657ec59912

Reply:
sorry, disregard the previous comment, i actually moved them back to `rethnet_rpc_server` in [THIS commit](https://github.com/NomicFoundation/hardhat/pull/3998/commits/f2fd69a0b195853f6019abac23e8ad3e6e22152c)